deployment/helm/skaha/sample-local-values.yaml

```diff
@@ -19,7 +24,7 @@ deployment:
 
     # The Resource ID of the Service that contains the Posix Mapping information
     posixMapperResourceID: "ivo://opencadc.org/posix-mapper"
-    # posixMapperResourceID: "ivo://cadc.nrc.ca/gms"
+    registryURL: http://reg.skaha-system.svc.cluster.local/reg
```

Comment:
To enforce users to set this themselves, this should be omitted.

Reply:
Done